packages/types/src/sales-channel/common.ts

```diff
@@ -1,3 +1,5 @@
+import { BaseFilterable } from "../dal"
```

Comment:
comment: This file, should we split the interfaces into read and mutations?

cc @olivermrbl

Reply:
Yes, so basically, take all mutational DTOs and move them to a separate file `mutations.ts` :)